Makefile

```diff
@@ -58,11 +59,30 @@ up: get vendor/hosts
 	@./bin/config.sh string SystemDNS container
 	@echo "NeoFS Developer Environment is ready"
 
+# Build up NeoFS
+.PHONY: up/basic
+up/basic: get vendor/hosts
+	@$(foreach SVC, $(START_NEOFS), $(shell docker-compose -f services/$(SVC)/docker-compose.yml up -d))
+	@./bin/tick.sh
+	@echo "Basic NeoFS Developer Environment is ready"
+
+# Build up certain service
+.PHONY: up/%
+up/%: get vendor/hosts
+	@docker-compose -f services/$*/docker-compose.yml up -d
+	@./bin/tick.sh
```

Comment:
I guess if you want to start certain service, you don't have to tick epoch. It is not possible for services before `morph_chain`, e.g. try `make up/basenet`.

Reply:
We suggest that before starting certain service with `make up/<service>`, basic services, including `morph_chain`, have been started already.